src/components/WhyTechSVG/WhyTech.js

```diff
@@ -7,17 +7,19 @@ const useStyles = makeStyles((theme) => ({
         fontFamily: 'Josefin Sans',
         fontStyle: 'normal',
         fontWeight: '600',
-        fontSize: '36px',
-        lineHeight: '57px',
+        fontSize: '28px',
+        lineHeight: '65px',
         display: 'flex',
         alignItems: 'center',
         color: '#000000',
         top: '856px',
+        textDecoration: 'underline',
+        textDecorationColor: '#FF7500',
     },
     cssBodyText: {
         fontFamily: 'poppins',
         fontStyle: 'normal',
-        fontWeight: 'normal',
+        fontWeight: '400',
```

Comment:
figma says font-weight should be '600', is there a reason it's set to 400?

Reply:
@hanlinc27 As for the font weight, not sure where you are seeing that. I see 600 for the header text and 400 for the body text on figma?